netbox_data_import/templates/netbox_data_import/import_preview.html

```
@@ -1003,6 +1075,24 @@ <h5 class="modal-title"><i class="mdi mdi-plus-circle"></i> Add Class Mapping</h
               <div class="mt-2" id="cm_role_search_results"></div>
             </div>
           </div>
+
+          <div id="cm_rack_type_row" style="display:none;">
+            <input type="hidden" name="rack_type_id" id="cm_rack_type_id" value="">
+            <div class="mb-3">
+              <label class="form-label">Rack type <span class="text-muted">(optional)</span></label>
+              <div class="input-group">
+                <input type="text" id="cm_rack_type_search_q" class="form-control" placeholder="Search NetBox rack types…">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)